packages/widgets/src/blocks/legacy-widget/edit/control.js

```diff
@@ -214,6 +217,18 @@ export default class Control {
 		}
 	}
 
+	/**
+	 * Perform a save when a multi widget's form is changed. Non-multi widgets
```

Comment:
Is "multi" and "non-multi" a recognised technical term? If not then we should probably add additional context here.